go.mod

```diff
@@ -37,6 +37,8 @@ require (
 	github.com/libp2p/go-libp2p-peerstore v0.1.3
 	github.com/libp2p/go-libp2p-swarm v0.2.2
 	github.com/mitchellh/go-homedir v1.1.0
+	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
+	github.com/modern-go/reflect2 v1.0.1 // indirect
```

Comment:
mind running `go mod tidy`? I'd like see if this gets removed. If this is intentional, where is this dep being used?

Reply:
Success, `go mod tidy` removed these